### PR TITLE
Feature/plcn 270 fix nested contact person overflow

### PR DESCRIPTION
**Pull request by s-bogelund**
Overflow er fixet, men det virker ikke helt som jeg gerne vil have det.

De forskellige NestedContactPerson items er stadig ikke præcis samme længde med en forskel på mellem 1-3px, men jeg har simpelthen ikke kunnet finde årsagen.

### CODE REVIEW

**Review by Nicolai-Bak — APPROVED** [review]

File: src/components/common/NestingIndicator.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ const NestingIndicator: FC<NestingIndicatorProps> = ({ height, onClick }) => {
 					cursor: 'pointer',
 				},
 			}}
+			{...rest}
```
Comment (itm-mne) [inline src/components/common/NestingIndicator.tsx:24]: Is ...rest necessary here if you don't pass the props any values?
Just curious :)

File: src/components/summaries/PersonInfoSummary.tsx [hunk 2]
```diff
@@ -65,14 +65,14 @@ const PersonInfoSummary: FC<PersonInfoSummaryInfoSummaryProps> = ({ persons }) =
 							}}
 						>
 							<Typography variant="note" sx={{ opacity: 0.7 }}>
-								{`${persons[0].firstName.charAt(0)} ${persons[0].lastName.charAt(0)}`}
+								{`${persons[0]?.firstName?.charAt(0)} ${persons[0]?.lastName?.charAt(0)}`}
```
Comment (itm-mne) [inline src/components/summaries/PersonInfoSummary.tsx:68]: If these values can be null, wouldn't the need a null-check?

File: src/components/summaries/PersonInfoSummary.tsx [hunk 3]
```diff
@@ -125,7 +125,7 @@ const PersonInfoSummary: FC<PersonInfoSummaryInfoSummaryProps> = ({ persons }) =
 							sx={{ width: 32, height: 32, bgcolor: theme.palette.primary.main }}
 						>
 							<Typography variant="note" sx={{ opacity: 0.7 }}>
-								{`${person.firstName.charAt(0)} ${person.lastName.charAt(0)}`}
+								{`${person?.firstName?.charAt(0)} ${person?.lastName?.charAt(0)}`}
```
Comment (itm-mne) [inline src/components/summaries/PersonInfoSummary.tsx:128]: Same as above

File: src/components/summaries/PersonInfoSummary.tsx [hunk 4]
```diff
@@ -145,7 +145,7 @@ const PersonInfoSummary: FC<PersonInfoSummaryInfoSummaryProps> = ({ persons }) =
 					>
 						<Avatar sx={{ width: 32, height: 32, bgcolor: theme.palette.primary.main }}>
 							<Typography variant="note" sx={{ opacity: 0.7 }}>
-								{`${person.firstName.charAt(0)} ${person.lastName.charAt(0)}`}
+								{`${person?.firstName?.charAt(0)} ${person?.lastName?.charAt(0)}`}
```
Comment (itm-mne) [inline src/components/summaries/PersonInfoSummary.tsx:148]: Same as above

**Review by itm-mne — APPROVED** [review]
General questions, no changes requested.

**Review by vla-ITM — APPROVED** [review]
Looks good! 

Some of the styling seems a little bit specific. Could be nice to maybe have those rely on the theme in the future. But otherwise looks nice! 👍

**Pull request merged**